src/Jobs/GenerateAnnotationPatch.php

```diff
@@ -359,13 +360,17 @@ protected function getSVGAnnotation($shapeId, $points)
         switch ($shapeId) {
             case Shape::pointId():
                 $radius = 1;
-                return new SVGCircle($points[0], $points[1], $radius);
+                return [new SVGCircle($points[0], $points[1], $radius)];
             case Shape::circleId():
-                return new SVGCircle($points[0], $points[1], $points[2]);
+                return [new SVGCircle($points[0], $points[1], $points[2])];
             case Shape::polygonId():
-                return new SVGPolygon($tuples);
+                return [new SVGPolygon($tuples)];
             case Shape::lineId():
-                return new SVGLine($points[0], $points[1], $points[2], $points[3]);
+                $lines = [];
+                for ($i = 0; $i < sizeof($points)-2; $i+=2) {
+                    $lines[] = new SVGLine($points[$i + 0], $points[$i + 1], $points[$i + 2], $points[$i + 3]);
```

Comment:
You could also use [SVGPolyline](https://github.com/meyfa/php-svg/blob/main/src/Nodes/Shapes/SVGPolyline.php) (see also [here](https://developer.mozilla.org/en-US/docs/Web/SVG/Tutorial/Basic_Shapes#polyline)). Then there is no need to return an array any more. Also, the drawn shape will probably look nicer.